package.json

```diff
@@ -49,6 +49,8 @@
         "@ngneat/dialog": "^5.1.0",
         "@ngneat/overview": "^5.1.1",
         "@ngneat/until-destroy": "^10.0.0",
+        "@ngrx/operators": "^18.0.1",
+        "@ngrx/signals": "^17.2.0",
```

Comment:
ngrx/signals has a v18 version (probably in beta, but we can use it already). Please update it.